src/scripts/main.js

```diff
@@ -1,5 +1,32 @@
 'use strict';
+console.log(document);
+const spider = document.querySelector('.spider');
+const parent = spider.offsetParent;
 
-document.addEventListener('click', (e) => {
-  // write code here
+let width = 0;
+let height = 0;
+
+const differenceWidth =
+  (document.querySelector('body').clientWidth - parent.clientWidth) / 2;
+const differenceHeight =
+  (document.querySelector('body').clientHeight - parent.clientHeight) / 2;
+console.log('differ:', differenceWidth, differenceHeight);
+  
+function getParam(param) {
+  if (param < 0) {
+    return 0
```

Comment:
There's a missing semicolon at the end of the `return 0` statement. This can lead to potential issues in JavaScript, so it's good practice to include it.